src/drivers/gps/gps.cpp

```diff
@@ -511,6 +567,73 @@ int GPS::setBaudrate(unsigned baud)
 	return 0;
 }
 
+bool GPS::fileExist(const char *filename)
+{
+	struct stat buffer;
+	return stat(filename, &buffer) == 0;
```

Comment:
@julianoes Can you file a Salesforce issue for stat? If they offer the API it not not crash in attempting to use the function.